logback-ecs-encoder/src/main/java/co/elastic/logging/logback/EcsEncoder.java

```diff
@@ -136,12 +138,21 @@ public byte[] encode(ILoggingEvent event) {
         return builder.toString().getBytes(UTF_8);
     }
 
+    private void serializeMessage(ILoggingEvent event, StringBuilder builder) {
+        if (messageLayout == null) {
+            EcsJsonSerializer.serializeFormattedMessage(builder, event.getFormattedMessage());
+        } else {
+            EcsJsonSerializer.serializeFormattedMessage(builder, this.messageLayout.doLayout(event));
```

Comment:
[minor/style] avoid unnecessary explicit `this` when not required.
```suggestion
            EcsJsonSerializer.serializeFormattedMessage(builder, messageLayout.doLayout(event));
```

Reply:
gah yes, do you want me to try and work this change through my process or will it be merged as is?

Reply:
> work this change through my process

What process are you referring to? Do you have a lengthy process from your company to go through for contributions?

This is just a minor change, so if it is to cumbersome for you we can merge as is or simply fix it ourselves.

Reply:
Fixed in my most recent commit.